src/plugins/data/common/field_formats/converters/url.ts

```diff
@@ -134,9 +134,10 @@ export class UrlFormat extends FieldFormat {
 
   textConvert: TextContextTypeConvert = value => this.formatLabel(value);
 
-  htmlConvert: HtmlContextTypeConvert = (rawValue, field, hit, parsedUrl) => {
+  htmlConvert: HtmlContextTypeConvert = (rawValue, field, hit, rawUrl) => {
     const url = escape(this.formatUrl(rawValue));
     const label = escape(this.formatLabel(rawValue, url));
+    const parsedUrl = rawUrl || this.params().parsedUrl;
```

Comment:
I do not like this solution. This is a bit confusing. If we decide to get the parsedUrl object from params, we need to pass it in this way from all places